Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift

```diff
@@ -96,7 +96,17 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
             guard let safe = try? Safe.getSelected() else {
                 return
             }
-            App.shared.ramper.startOnRamp(safe: safe)
+
+            let vc = SelectTopUpAddressViewController()
```

Comment:
Does it make sense to de-duplicate this? Looks like it is the same as in AssetsViewController.

Reply:
Will do in the next pr